ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/management-tech-support.j2

```diff
@@ -0,0 +1,42 @@
+{# j2lint: disable=S2 #}
+{# Management Tech-Support #}
+{% if management_tech_support is arista.avd.defined %}
+
+## Management Tech-Support
+{%     if management_tech_support.policy_show_tech_support is arista.avd.defined %}
+
+### Policy
+{%         if management_tech_support.policy_show_tech_support.exclude_commands is arista.avd.defined %}
+
+#### Exclude Commands
+
+| Command | Type |
+| ------- | ---- |
+{%             if management_tech_support.policy_show_tech_support.exclude_commands is arista.avd.defined %}
+{%                 for exclude_command in management_tech_support.policy_show_tech_support.exclude_commands %}
+| {{ exclude_command.command | replace('|', '\|') }} | {{ exclude_command.type | arista.avd.default('text') }} |
+{%                 endfor %}
+{%             endif %}
+{%         endif %}
+
```

Comment:
Please delete this blank line.

Reply:
resolved in [7b1f5ba](https://github.com/aristanetworks/ansible-avd/pull/1618/commits/7b1f5ba42c6af20d5e5acdbe2d554e1b1df904cd)